src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -304,6 +310,13 @@ const Notebooks = _.flow(
               this.refresh()
             }
           }),
+          exportingNotebookName && h(ExportNotebookModal, {
+            printName: printName(exportingNotebookName),
+            thisWorkspaceId: workspaceId,
+            thisWorkspaceNamespace: namespace,
+            thisWorkspaceName: name, bucketName,
```

Comment:
TOL: Given how much of the workspace info we need to pass in here, I wonder if it would be cleaner to pass the whole workspace object.